servlet-feature-pack/src/main/resources/content/README.txt

```diff
@@ -18,9 +18,9 @@ Key Features
 
 Release Notes
 -------------
-You can obtain the 10.0.0.Final release notes here:
+You can obtain the 14.0.1.Final release notes here:
 
-http://wildfly.org/news/2016/01/29/WildFly10-Released/
+http://wildfly.org/news/2018/08/30/WildFly14-Final-Released/
```

Comment:
master is 15 now... so a different link probably.

Reply:
Well, it isn;t released yet - chicken and egg

Reply:
Probably just change the link to something meaningful then.